admin/app/purge/CdnPurge.scala

```diff
@@ -13,20 +13,19 @@ class CdnPurge(wsClient: WSClient) extends ExecutionContexts with Dates with Log
   private val serviceId = "2eYr6Wx3ZCUoVPShlCM61l"
 
   // Performs soft purge which will still serve stale if there is an error
-  def soft(key:String): Future[Boolean] = {
+  def soft(key:String): Future[WSResponse] = {
     // under normal circumstances we only ever want this called from PROD.
-    // Don't want too much decache going on.
+    // Don't want too much decaching going on.
     if (environment.isProd) {
       val purgeRequest = wsClient.url(s"https://api.fastly.com/service/$serviceId/purge/$key")
         .withHeaders("Fastly-Key" -> fastly.key,
                      "Fastly-Soft-Purge" -> "1")
         .post("")
 
-      purgeRequest.foreach(r => log.info(s"purge $key from Fastly with response ${r.statusText}"))
-      purgeRequest.map(_.status == 200)
+      purgeRequest.onSuccess { case r => log.info(s"purge $key from Fastly with response ${r.statusText}") }
+      purgeRequest
```

Comment:
Are there any status codes that we would consider a failure that would not produce `Future.failed`?   If so we will end up showing a success message in those cases I think.